src/thor/triplegbuilder.cc

```diff
@@ -689,6 +723,7 @@ TripLeg_Edge* AddTripEdge(const AttributesController& controller,
 
   // Set speed if requested
   if (controller.attributes.at(kEdgeSpeed)) {
+    // TODO: if this is a transit edge then the costing will throw
```

Comment:
Is this intended to be fixed in this PR or is it a caveat for future us?

Reply:
its been broken for a while and the fix is non-trivial. i think we should fix this in a subsequent pr (this happens in another spot at well)